library/model/src/test/kotlin/org/cru/godtools/model/LanguageTest.kt

```diff
@@ -63,5 +64,13 @@ class LanguageTest {
             }
         }
     }
+
+    @Test
+    fun `getForceDisplayName()`() {
+        val language = parseJson("language_force_name.json")
```

Comment:
for models we've been keeping parsing tests independent from functionality tests. That makes it easier in the future when the test fails to determine is it just parsing that's failing, or is it the name fallback behavior that is failing.